tests/lib/rules/no-debug.test.ts

```diff
@@ -419,6 +426,20 @@ ruleTester.run(RULE_NAME, rule, {
         },
       ],
     },
+    {
+      code: `
+        import { screen } from '@testing-library/dom'
+        screen.logTestingPlaygroundURL()
+      `,
+      options: [{ utilNames: ['logTestingPlaygroundURL'] }],
+      errors: [
+        {
+          line: 3,
+          column: 16,
+          messageId: 'noDebug',
+        },
+      ],
+    },
```

Comment:
Can you add another test exactly like this one but without setting `options` so we test `logTestingPlaygroundURL` is reported by default?